Bottom: c6d3371aa54fcf7d526cc4ab9cb007e5dbded790
Top:    fcb939492941fdc54d1ffa413b4a39b976d4aebb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:53 -0500

update worker configure like master


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 6232036e..c18ebba4 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -146,10 +146,10 @@ c['buildbotNetUsageData'] = 'basic'
 
 import openembedded
 reload(openembedded)
-c['workers'].extend(openembedded.workers)
-c['schedulers'].extend(openembedded.schedulers)
-c['builders'].extend(openembedded.builders)
-c['change_source'].extend(openembedded.change_source)
+c['workers'].extend(openembedded.WorkerConfig['workers'])
+c['schedulers'].extend(openembedded.WorkerConfig['schedulers'])
+c['builders'].extend(openembedded.WorkerConfig['builders'])
+c['change_source'].extend(openembedded.WorkerConfig['change_source'])
 
 import ptxdist
 reload(ptxdist)
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index ca79dca1..c946d579 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -38,6 +38,7 @@ c['workers'] = [
 
 DEFAULT_REPO = 'git@git.novatech-llc.com:ntel/setup-scripts.git'
 
+
 # CHANGESOURCES
 c['change_source'] = [
     changes.GitPoller(
